lib/cretonne/src/ir/dfg.rs

```diff
@@ -657,6 +665,28 @@ impl DataFlowGraph {
                         })
     }
 
+    /// Removes the argument `val` from `ebb`. Returns true if the removal is sucessful.
+    pub fn remove_ebb_arg(&mut self, ebb: Ebb, val: Value) -> bool {
```

Comment:
I have a semantics problem here. If I want to update the value table after the removal of the Ebb arg, by what should I replace the `ValueData::Arg::...`? Should I transform the method into `remove_ebb_arg_and_change_to_alias`, which is my use case here?